en/components/autocomplete.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Autocomplete Directive
+_description: The igxAutocomplete directive provides a way to enhance a text input by showing a panel of suggested options provided by the developer.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Autocomplete components, Angular Autocomplete directives, Angular Autocomplete controls
+---
+
+## Autocomplete
+<p class="highlight">
+
+The [**igxAutocomplete**]({environment:angularApiUrl}/classes/igxautocompletedirective.html) directive provides a way to enhance a text input by showing a [drop down]({environment:angularApiUrl}/classes/igxdropdowncomponent.html) with suggested options provided by the developer.
+
+The [igx-autocomplete]({environment:angularApiUrl}/classes/igxautocompletedirective.html) suggestions will show as long as you start typing in the text input. By default, the first item is always highlighted, so that it can be easily selected using the Enter key. The other way is to click an item from the list and the input value will be automatically updated and the drop down will be automatically closed, unless the onItemSelected event is handled and canceled. 
```

Comment:
replace as long as you - with "will show once you start typing in the input

Reply:
update "highlighted, so it can be easily selected.."

Reply:
Other way of selecting an item is by clicking on it..., Once the item is selected the input value will .....
Preventing the selection and auto close of the drop-down could be achieved with inItemSelceted event cancellation.